aspnetcore/security/authorization/limitingidentitybyscheme.md

```diff
@@ -41,102 +27,51 @@ In the preceding code, two authentication handlers have been added: one for cook
 
 > [!NOTE]
 > Only one JWT bearer authentication is registered with the default authentication scheme `JwtBearerDefaults.AuthenticationScheme`. Additional authentication has to be registered with a unique authentication scheme.
 
-The next step is to update the default authorization policy to accept both authentication schemes. For example:
+Update the default authorization policy to accept both authentication schemes. For example:
 
-```csharp
-public void ConfigureServices(IServiceCollection services)
-{
-    // Code omitted for brevity
-
-    services.AddAuthorization(options =>
-    {
-        var defaultAuthorizationPolicyBuilder = new AuthorizationPolicyBuilder(
-            JwtBearerDefaults.AuthenticationScheme,
-            "AzureAD");
-        defaultAuthorizationPolicyBuilder = 
-            defaultAuthorizationPolicyBuilder.RequireAuthenticatedUser();
-        options.DefaultPolicy = defaultAuthorizationPolicyBuilder.Build();
-    });
-}
-```
+[!code-csharp[](~/security/authorization/limitingidentitybyscheme/samples/AuthScheme/Program.cs?name=snippet_ma&highlight=20-29)]
 
 As the default authorization policy is overridden, it's possible to use the `[Authorize]` attribute in controllers. The controller then accepts requests with JWT issued by the first or second issuer.
 
 See [this GitHub issue](https://github.com/dotnet/aspnetcore/issues/26002) on using multiple authentication schemes.
+
+The following example uses [Azure Active Directory B2C](/azure/active-directory-b2c/overview) and another [Azure Active Directory](/azure/active-directory/authentication/overview-authentication) tenant:
+
+[!code-csharp[](~/security/authorization/limitingidentitybyscheme/samples/AuthScheme/Program.cs?name=snippet_ma2&highlight=9-49)]
+
+In the preceding code, <xref:Microsoft.AspNetCore.Authentication.AuthenticationSchemeOptions.ForwardDefaultSelector> is used to select a default scheme for the current request that authentication handlers should forward all authentication operations to by default. The default forwarding logic checks the most specific <xref:Microsoft.AspNetCore.Authentication.AuthenticationSchemeOptions.ForwardAuthenticate>, <xref:Microsoft.AspNetCore.Authentication.AuthenticationSchemeOptions.ForwardChallenge>, <xref:Microsoft.AspNetCore.Authentication.AuthenticationSchemeOptions.ForwardForbid>, <xref:Microsoft.AspNetCore.Authentication.AuthenticationSchemeOptions.ForwardSignIn>, and <xref:Microsoft.AspNetCore.Authentication.AuthenticationSchemeOptions.ForwardSignIn> setting first, followed by checking the <xref:Microsoft.AspNetCore.Authentication.AuthenticationSchemeOptions.ForwardDefaultSelector>, followed by <xref:Microsoft.AspNetCore.Authentication.AuthenticationSchemeOptions.ForwardDefault>. The first non null result is used as the target scheme to forward to. For more information, see <xref:security/authentication/policyschemes>.
```

Comment:
@HaoK  See [AuthenticationSchemeOptions.ForwardDefaultSelector](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.authentication.authenticationschemeoptions.forwarddefaultselector?view=aspnetcore-6.0) 
It lists Challenge/Forbid/SignIn/SignOut setting first
but it should be `ForwardChallenge`, `ForwardForbid`, `ForwardSignIn`, and `ForwardSignOut` setting first
They dropped the ***FORWARD*** prefix, right?

Reply:
Added the links you requested.

Reply:
Yep correct, Forward is a prefix for all of those settings